realsense2_camera/launch/rs_launch.py

```diff
@@ -95,35 +94,35 @@ def yaml_to_dict(path_to_yaml):
 def launch_setup(context, params, param_name_suffix=''):
     _config_file = LaunchConfiguration('config_file' + param_name_suffix).perform(context)
     params_from_file = {} if _config_file == "''" else yaml_to_dict(_config_file)
+
+    # Overwrite the params with the values provided in YAML config file
+    params.update(params_from_file)
+
     # Realsense
     if (os.getenv('ROS_DISTRO') == "dashing") or (os.getenv('ROS_DISTRO') == "eloquent"):
         return [
             launch_ros.actions.Node(
-                package='realsense2_camera',
-                node_namespace=LaunchConfiguration('camera_namespace' + param_name_suffix),
-                node_name=LaunchConfiguration('camera_name' + param_name_suffix),
-                node_executable='realsense2_camera_node',
-                prefix=['stdbuf -o L'],
-                parameters=[params
-                            , params_from_file
-                            ],
-                output='screen',
-                arguments=['--ros-args', '--log-level', LaunchConfiguration('log_level' + param_name_suffix)],
+                package         = 'realsense2_camera',
+                node_namespace  = params['camera_namespace' + param_name_suffix],
+                node_name       = params['camera_name' + param_name_suffix],
```

Comment:
Camera name and namespace values got from LaunchConfiguration are already stored in 'params'. Also, those values will get overwritten if they are defined in YAML config file.